crates/cxx-qt-build/src/lib.rs

```diff
@@ -769,6 +829,12 @@ impl CxxQtBuilder {
                 // RCC file.
                 .file(qml_module_registration_files.rcc);
 
+            // Add any include paths the qml module registration needs
+            // this is most likely the moc folder for the plugin
+            if let Some(include_path) = qml_module_registration_files.include_path {
+                self.cc_builder.include(include_path);
```

Comment:
Isn't this redundant with the include paths that are added in the loop?

Reply:
I guess the fun is what if the different moc calls ended up in different directories? Maybe i can make a HashSet of all the possible directories (which are likely to be one at the moment but could change if we modified our moc call). Then it could add that HashSet of dirs to the include (avoiding duplicates)